packages/instantsearch-ui-components/src/components/chat/ChatPrompt.tsx

```diff
@@ -1,9 +1,14 @@
 /** @jsx createElement */
 
+import { useRef, useLayoutEffect, useCallback } from 'react';
```

Comment:
these can't be used. Is there some way to hoist the hooks outside the chat component, like done for scroll?

Reply:
Done!